src/scene/shader-lib/chunks/common/vert/transform.js

```diff
@@ -59,13 +59,14 @@ mat4 getModelMatrix() {
 #ifdef DISPLACEMENT
 uniform sampler2D texture_heightMap;
 uniform float material_heightMapFactor;
+uniform float material_displacementOffset;
 #endif
 vec4 getPosition() {
     dModelMatrix = getModelMatrix();
 
     #ifdef DISPLACEMENT
     vec4 dMapSample = texture2D(texture_heightMap, vertex_texCoord0.xy);
-    float displacer = dMapSample.r * material_heightMapFactor * 10.;
+    float displacer = dMapSample.r * material_heightMapFactor * 10. + material_displacementOffset;
```

Comment:
I'm not sure about the `*10 ` part here?

Reply:
That can be removed when `material_heightMapFactor` is replaced with new attribute.

Reply:
Wouldn't it be more useful for vertex displacement to be implemented as an overridable shader chunk instead?

Reply:
I am not sure, you are the guys to call the shot.